README.rst

```diff
@@ -1,132 +1,196 @@
-You might also want to visit `our website <http://coala-analyzer.org/>`_.
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala: Language Independent Code Analysis
+-------------
+**coala provides a common command-line interface for linting and fixing all
+your code, regardless of the programming languages you use.**
+
+With coala, users can create
+`rules and standards <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
```

Comment:
not sure we should point users there, CC @Adrianzatreanu maybe even remove that doc page? Or make it hold better info with useful examples.